### PR TITLE
Use global db in travis

### PR DESCRIPTION
**Pull request by phadej**
Without sandbox, there are less moving parts, and tests seem to be more robust.

### CODE REVIEW

**Comment by phadej** [comment]
ping @23Skidoo, is this ok? `Cabal`'s `package-tests` seem to be more robust

E.g. https://travis-ci.org/phadej/cabal/builds/86029580 (older GHC fail because of `-Werror`)

File: .travis.yml [hunk 1]
```diff
@@ -27,23 +28,20 @@ install:
 # ./dist/setup/setup here instead of cabal-install to avoid breakage when the
 # build config format changed.
 script:
- - cabal sandbox init
+ # We depend on parsec nowadays, which isn't distributed with GHC <8.0
```
Comment (23Skidoo) [inline .travis.yml]: May want to make this conditional to make sure that Cabal can be built with stock GHC 8.0.

**Comment by phadej** [comment]
ping @23Skidoo Made parsec installation configurable thru environment variable (which could be set for GHC which provide parsec)

**Comment by 23Skidoo** [comment]
Merged, thanks.

**Comment by 23Skidoo** [comment]
BTW, you're using the user db, not the global one.

**Comment by phadej** [comment]
@23Skidoo, ah sorry, have to be more accurate in the terminology, here *global ≡ not sandbox*

**Pull request merged**